js/rpg_objects/Game_System.js

```diff
@@ -118,7 +118,7 @@ Game_System.prototype.saveCount = function() {
 };
 
 Game_System.prototype.frameCount = function() {
-    return this._frameCount;
+    return SceneManager.frameCount();
 };
```

Comment:
this function may no longer be required, because it does not originally exist ;-)

Reply:
All right, I will remove this.